Mauvais message d'erreur quand on lance le serveur en dehors de l'environnement virtuel

Voici l'erreur que j'ai quand le lance `make run-back` sans avoir lancé `source zdsenv/bin/activate` auparavant. Il ne faut pas que le message `Your environment does not have django, please run pip install -r requirements-dev.txt` s'affiche car c'est trompeur ; il faut que le message ``No virtual environment detected, usually contributors use virtual environment 'zdsenv' with `source zdsenv/bin/activate` ``s'affiche au préalable !

```
zmd is running
python manage.py runserver
('\x1b[36;1mINFO: No virtual_env detected,', 'usually contributors use virtual environment `zdsenv`\x1b[00;0m')
Your environment does not have django, please run pip install -r requirements-dev.txt
Makefile:30: recipe for target 'run-back' failed
make: *** [run-back] Error 1
```

--- a/manage.py
+++ b/manage.py
@@ -35,10 +35,10 @@ if __name__ == '__main__':
         import django
     except ImportError:
         if os.environ.get('virtual_env') is None:
-            print('\033[36;1mINFO: No virtual_env detected,',
+            print('\033[36;1mINFO: No virtual_env detected, '
             'usually contributors use virtual environment `zdsenv`\033[00;0m')
         elif os.path.normpath(os.environ.get('virtual_env')) != os.path.normpath('./zdsenv'):
-            print('\033[36;1mINFO: The current virtual_env is `%s`,',
+            print('\033[36;1mINFO: The current virtual_env is `%s`, '
             'usually contributors use `./zdsenv`\033[00;0m' % os.environ.get('virtual_env'))
 
         print("\033[31;1mYour environment does not have django, please run pip install -r requirements-dev.txt\033[00;0m")

--- a/manage.py
+++ b/manage.py
@@ -34,12 +34,12 @@ if __name__ == '__main__':
     try:
         import django
     except ImportError:
-        if os.environ.get('virtual_env') is None:
-            print('\033[36;1mINFO: No virtual_env detected, '
+        if os.environ.get('VIRTUAL_ENV') is None:
+            print('\033[36;1mINFO: No VIRTUAL_ENV detected, '
             'usually contributors use virtual environment `zdsenv`\033[00;0m')
-        elif os.path.normpath(os.environ.get('virtual_env')) != os.path.normpath('./zdsenv'):
-            print('\033[36;1mINFO: The current virtual_env is `%s`, '
-            'usually contributors use `./zdsenv`\033[00;0m' % os.environ.get('virtual_env'))
+        elif os.path.normpath(os.environ.get('VIRTUAL_ENV')) != os.path.normpath('./zdsenv'):
+            print('\033[36;1mINFO: The current VIRTUAL_ENV is `%s`, '
+            'usually contributors use `./zdsenv`\033[00;0m' % os.environ.get('VIRTUAL_ENV'))
 
         print("\033[31;1mYour environment does not have django, please run pip install -r requirements-dev.txt\033[00;0m")
         sys.exit(1)

